packages/components/styles/wp-button-override.scss

```diff
@@ -0,0 +1,21 @@
+@use './_wp-components-theme-variables' as theme;
+
+@import '@wordpress/base-styles/colors';
+@import '@wordpress/base-styles/variables';
+
+.a8c-components-wp-button {
+	&.is-secondary:not([aria-pressed="true"]) {
+		color: theme.$components-color-foreground;
```

Comment:
@ciampo Are the `$components-color-` variables too much? They're not strictly necessary but I thought some consistency here could be helpful since we're seeing some ad hoc CSS variable-based theming starting to happen.